alpha/alpha-server/src/main/java/org/apache/servicecomb/saga/alpha/server/tcc/OmegaCallbacksRegistry.java

```diff
@@ -32,6 +33,10 @@
 
   private final static Map<String, Map<String, OmegaCallback>> REGISTRY = new ConcurrentHashMap<>();
```

Comment:
Please don't use the static Map as the registry, it could introduce lots of trouble here.

Reply:
i think concurrent problem don't exist